x/msgfees/client/cli/tx.go

```diff
@@ -153,73 +134,64 @@
 
 func GetUpdateNhashPerUsdMilProposal() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "nhash-per-usd-mil <title> <description> <nhash-per-usd-mil> <deposit>",
+		Use:     "nhash-per-usd-mil <nhash-per-usd-mil>",
 		Aliases: []string{"npum", "n-p-u-m"},
-		Args:    cobra.ExactArgs(4),
+		Args:    cobra.ExactArgs(1),
```

Comment:
The `Use` defines the command without respect to the parent command. It's not correct to name this command `msgfees` because it's already in a `msgfees` parent command, and this specific command should be named `nhash-per-usd-mil`.